src/client/game.cpp

```diff
@@ -4359,15 +4359,15 @@ void Game::showPauseMenu()
 #if __IOS__
 	ypos += 0.5f;
 #endif
-	std::ostringstream os;
+	std::ostringstream os; // ToDo: ".x2" for buttons
```

Comment:
@luk3yx any idea how is best way to  implement this?
```lua
local high_dpi = (PLATFORM == "OSX" and tonumber(core.settings:get("screen_dpi")) / 72 >= 2) or
		core.get_screen_info().density >= 3
```

Reply:
I've added a `Game::isHighDpi()` method